components/starratingcontentnode.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version = "1.0" encoding = "utf-8" ?>
+<component name="StarRatingContentNode" extends="ContentNode">
+    <children>
+        <ContentNode>
+            <ContentNode 
+                HDPosterUrl = "pkg:/images/rate-star.png" />
+            <ContentNode 
+                HDPosterUrl = "pkg:/images/rate-star.png" />
+            <ContentNode 
+                HDPosterUrl = "pkg:/images/rate-star.png" />
+            <ContentNode 
+                HDPosterUrl = "pkg:/images/rate-star.png" />
+            <ContentNode 
+                HDPosterUrl = "pkg:/images/rate-star.png" />
+        </ContentNode>
```

Comment:
Can you create a code file for this and create the row inside of that?
It looks messy in this way because it looks like [duplicated code](https://refactoring.guru/smells/duplicate-code)